cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -97,11 +98,12 @@ public void resetServer() {
      * @param userAnnotation An annotation to be applied after the reconcile has started, for example "approve" or "stop".
      * @param rebalanceSpec A custom rebalance specification. If null a blank spec will be used.
      * @return A KafkaRebalance instance configured with the supplied parameters.
+     * @return Reason for the condition
```

Comment:
I think it's wrong. You have got two `@return` I guess you meant to add a `@param reason`

Reply:
Oops Sorry, Let me correct it

Reply:
Fixed